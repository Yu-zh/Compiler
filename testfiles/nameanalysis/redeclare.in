int funa() {
    int a = 1;
    int a = 2;
    int b = 3;
}
int funa() {}
int read() {}
