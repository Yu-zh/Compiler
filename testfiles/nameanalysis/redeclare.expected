Error at line 3: symbol 'a' is already declared!
Error at line 6: symbol 'funa' is already declared!
Error at line 7: symbol 'read' is already declared!